documenation/documentation.html

```diff
@@ -926,8 +928,245 @@ <h3 class="ternary-text">Card structure</h3>
           </div>
         </div>
 
-        <div id="image">Image</div>
-        <div id="input">Input</div>
+        <div id="image">
+          <h2>Images</h2>
+          <p>
+            Images are the important aspects of any website. HTML provides us
+            <span class="primary-text">img</span> tag with help of that we can
+            put images on web page.
+          </p>
+          <hr />
+          <p class="title">Responsive Image</p>
+          <p>
+            Responsive Image is a image which will have width and height
+            inherited from its parents element.Responsive Images are used to
+            dsiplay Images on websites, which are screen responsive.
+          </p>
+          <div class="image-container classic-bg-color pd-1">
+            <!-- use class responsive-img on img tag to make it responsive -->
+            <img class="responsive-img" src="https://picsum.photos/500" />
+          </div>
+          <div class="code-box">
+            <iframe
+              src="https://www.thiscodeworks.com/embed/61fd40bf33c09d0015b44d93"
+              style="width: 100%; height: 125px"
```

Comment:
you can move the iFrame styling to one of the common css files.

Reply:
It is default coming from ifram i am using. and It varies code snippet to snippet. So, its hard to give static heights and width to this iframes. But thanks for suggesting

Reply:
You could maybe give it a fixed height and width. In that way your code snippets will look uniform throughout the website. Again, this is just a suggestion, you can leave it as is if you want to.